test/unit/specs/lcdClient.spec.js

```diff
@@ -1,229 +1,459 @@
-      [`${remoteLcdURL}/stake/delegators/abc/txs?type=bonding`, undefined],
-      [`${remoteLcdURL}/stake/delegators/abc/txs?type=unbonding`, undefined],
-      [`${remoteLcdURL}/stake/delegators/abc/txs?type=redelegate`, undefined]
-    ])
-  })
+      it(`updateDelegations`, async () => {
+        axios.post = jest.fn().mockReturnValue({})
+        await client.updateDelegations(`abc`)
+        expect(axios.post.mock.calls[0]).toEqual([
+          `http://localhost:${
+            mockServer.address().port
+          }/stake/delegators/abc/delegations`,
+          undefined
+        ])
+      })
 
-  it(`queries for undelegations between a delegator and a validator`, async () => {
-    axios.get = jest.fn().mockReturnValue({})
-    await client.queryUnbonding(`abc`, `def`)
-    expect(axios.get.mock.calls[0]).toEqual([
-      `${remoteLcdURL}/stake/delegators/abc/unbonding_delegations/def`,
-      undefined
-    ])
-  })
+      it(`queries for staking parameters`, async () => {
+        axios.get = jest.fn().mockReturnValue({})
+        await client.getParameters()
+        expect(axios.get.mock.calls[0]).toEqual([
+          `${remoteLcdURL}/stake/parameters`,
+          undefined
+        ])
+      })
 
-  it(`queries for a validator`, async () => {
-    axios.get = jest.fn().mockReturnValue({})
-    await client.getCandidate(`abc`)
-    expect(axios.get.mock.calls[0]).toEqual([
-      `${remoteLcdURL}/stake/validators/abc`,
-      undefined
-    ])
-  })
+      it(`queries for staking pool`, async () => {
+        axios.get = jest.fn().mockReturnValue({})
+        await client.getPool()
+        expect(axios.get.mock.calls[0]).toEqual([
+          `${remoteLcdURL}/stake/pool`,
+          undefined
+        ])
+      })
+    })
 
-  it(`queries for staking parameters`, async () => {
-    axios.get = jest.fn().mockReturnValue({})
-    await client.getParameters()
-    expect(axios.get.mock.calls[0]).toEqual([
-      `${remoteLcdURL}/stake/parameters`,
-      undefined
-    ])
-  })
+    describe(`queryAccount`, () => {
+      it(`returns an account`, async () => {
+        axios.get = jest.fn().mockReturnValueOnce(
+          Promise.resolve({
+            data: {
+              value: {
+                type: `auth/Account`,
+                value: {
+                  address: `cosmosaccaddr1xr9rxc3jr6xzzugreykyrdsyjhtx8qvm4et3yp`,
+                  coins: [{ denom: `steak`, amount: `10` }],
+                  public_key: null,
+                  account_number: `768`,
+                  sequence: `0`
+                }
+              }
+            }
+          })
+        )
 
-  it(`queries for staking pool`, async () => {
-    axios.get = jest.fn().mockReturnValue({})
-    await client.getPool()
-    expect(axios.get.mock.calls[0]).toEqual([
-      `${remoteLcdURL}/stake/pool`,
-      undefined
-    ])
-  })
+        expect(await client.queryAccount(`address`)).toEqual({
+          type: `auth/Account`,
```

Comment:
I think this is a bug in the LCD and should only return the value not the type. Not a problem here. I file an issue.

Reply:
https://github.com/cosmos/cosmos-sdk/issues/2503